src/plugins/workspace/public/utils.ts

```diff
@@ -70,20 +70,29 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
 };
 
 // Get all apps excluding management category
-export const getAllExcludingManagementApps = (applications: PublicAppInfo[]): PublicAppInfo[] => {
+export const getAllIncludingDashboardsManagementApps = (
+  applications: PublicAppInfo[]
+): PublicAppInfo[] => {
   return applications.filter(
-    ({ navLinkStatus, chromeless, category }) =>
-      navLinkStatus !== AppNavLinkStatus.hidden &&
-      !chromeless &&
-      category?.id !== DEFAULT_APP_CATEGORIES.management.id
+    ({ navLinkStatus, chromeless, category, workspaceAccessibility, id }) => {
+      const filterCondition =
+        navLinkStatus !== AppNavLinkStatus.hidden &&
+        !chromeless &&
+        workspaceAccessibility !== WorkspaceAccessibility.NO;
+      if (category?.id === DEFAULT_APP_CATEGORIES.management.id) {
+        return filterCondition && id === 'management';
+      } else {
+        return filterCondition;
+      }
```

Comment:
```suggestion

      return filterCondition;
```
Nit: better to remove useless `else`.

Reply:
Have deleted `else`